tests/forms/test_public.py

```diff
@@ -9,10 +9,11 @@
 
 
 def test_login_form_validate_success(user):
-    """Login successful."""
+    """Login successful (irrespective of casing)."""
     user.set_password('example')
     user.save()
-    form = LoginForm(username=user.email, password='example')
+    username_with_different_casing = user.email.upper()
+    form = LoginForm(username=username_with_different_casing, password='example')
```

Comment:
@mxtthias Back to my account! So, one might argue that this should be a separate test case, not an extension to an existing one. I.e. "test that you can login with exactly correct input" and "test that you can login with *essentially* correct input", respectively.

In my thinking, it's nice to have simple and neat functions. But I'm a bit averse taking it to the extreme... I'm thinking it will be **a lot** of functions if each and everyone can have only a few lines, making it difficult to overlook. Do we have some general policy on McCabe complexity or *instructions-within-a-function*? If so, it should be pretty simple to add code style checks for in Jenkins (and good guidance going forward).

Anyhow, please let me know if you disagree!